models/junos/security_policy/junos_security_policies.yml

```diff
@@ -0,0 +1,514 @@
+---
+GENERATOR_VERSION: '1.0'
+ANSIBLE_METADATA: |
+  {
+     'metadata_version': '1.1',
+     'status': ['preview'],
+     'supported_by': 'network'
+  }
+NETWORK_OS: junos
+RESOURCE: l3_interfaces
+COPYRIGHT: Copyright 2022 Red Hat
+LICENSE: gpl-3.0.txt
+
+DOCUMENTATION: |
+  module: junos_l3_interfaces
+  version_added: 2.12
+  short_description: Manage security policies on Juniper JUNOS devices
+  description: This module provides declarative management of security policies on Juniper JUNOS devices
+  author: Pranav Bhatt (pbhatt@redhat.com)
+  requirements:
+    - ncclient (>=v0.6.4)
+  notes:
+    - This module requires the netconf system service be enabled on the device being managed.
+    - This module works with connection C(netconf). See L(the Junos OS Platform Options,../network/user_guide/platform_junos.html).
```

Comment:
Irrelevant to the context of this PR: this needs an update to FQCN, as this is supposed to break use U(full URL instead)